src/main/kotlin/no/nav/faktureringskomponenten/controller/mapper/ModelDtoMapperExtensions.kt

```diff
@@ -61,7 +62,7 @@ private val FullmektigDto.tilFullmektig: Fullmektig
 private val Faktura.tilResponseDto: FakturaResponseDto
     get() = FakturaResponseDto(
         datoBestilt = this.datoBestilt,
-        sistOppdatert = this.sistOppdatert,
+        sistOppdatert = LocalDate.ofInstant(this.endretTidspunkt, ZoneId.systemDefault()),
```

Comment:
Kan vurdere å rename denne til "mer riktig" endretTidspunkt, men tenkte det var godt nok å beholde gamle navnet for å slippe å oppdatere i web også. Men om det er uenighet så kan jeg få fikset

Reply:
Greit for meg